cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,77 @@
+from email.mime.multipart import MIMEMultipart
+from email.utils import COMMASPACE, formatdate
+from smtplib import SMTP, SMTPException
+import logging
+
+""" Report email notification for anomalies """
+
+# Define module-level logger.
+_log = logging.getLogger(__name__)
+
+class EmailStore():
+
+    def __init__(self, host, port, subject, to, sender, body):
+        self.host = host
+        self.port = port
+        self.subject = subject
+        self.to = to
+        self.sender = sender
+        self.body = body
+
+    def write(self, record):
+        """Write JSON records to the file system.
```

Comment:
This `write` method does nothing, but the docstring says it writes the JSON records to a file.

Reply:
Changed the docstring.